databases/relational-databases.md

```diff
@@ -130,7 +130,7 @@ Because you must define a schema to store data in a relational database, some de
   - Create another table that is different but related, and that shares at least one column with the first table created (i.e. create a table that can be logically `JOIN`ed to the first table)
   - Add rows of data to the new table(s)
   - Execute a few queries to retrieve data, as guided by instructor or volunteers. (While this will likely be review of content from the [SQL](https://github.com/Techtonica/curriculum/blob/master/SQL/sql.md) lesson plan, practice will help sharpen these skills.)
```

Comment:
@raibove you can made the link (https://github.com/Techtonica/curriculum/blob/master/SQL/sql.md) short like (../SQL/sql.md)

Reply:
@Kundan28  if I use (../SQL/sql.md) then it takes me to (https://github.com/raibove/curriculum/blob/raibove-relational-database/databases/sql.md). Since I am working from forked repo. What should I do??

Reply:
Would you please tell me if this link exist or not ?https://github.com/Techtonica/curriculum/blob/master/SQL/sql.md 
As it's showing me 404 error

Reply:
It showed me same error I changed the link but cannot make it short

Reply:
Hi Shweta, Kundan was just trying to give you an example. Change it to /databases/sql.md.

Reply:
> Hi Shweta, Kundan was just trying to give you an example. Change it to /databases/sql.md.

I made the change @alodahl